proxy/src/http.rs

```diff
@@ -66,6 +67,7 @@ where
     let store = Arc::new(RwLock::new(store));
     let subscriptions = crate::notification::Subscriptions::default();
 
+    let account_filter = path("accounts").and(account::filters(&Arc::clone(&registry)));
```

Comment:
I would keep `path("accounts")` in `account.rs` so it is consistent with the other declarations of the path

Reply:
That was @xla. I would prefer that too.

Reply:
We gonna move all of the other ones out. It's prefactoring, the great promise of a golden future to come.

Reply:
Ok. That wold cut the duplication that we currently have setting filters for testing purposes :ok_hand: